README.rst

```diff
@@ -0,0 +1,174 @@
+AlloyDB for PostgreSQL for LangChain
+==================================================
+
+- |preview| |pypi| |versions|
+
+- `Client Library Documentation`_
+- `Product Documentation`_
+- `API Reference`_
```

Comment:
Remove API reference. Use API reference link for Client Library Documentation instead